virttest/utils_kernel_module.py

```diff
@@ -0,0 +1,188 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2020
+# Author: Sebastian Mitterle <smitterl@redhat.com>
+
+"""
+Handle Linux kernel modules
+"""
+
+import os
+import logging
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+
+def reload(module_name, force, params):
```

Comment:
Actually reading the name here, the `reload_if_needed` might be a better name as that is the difference between creating KernelModuleHandler and calling it's reload... Let's leave it on you, I'm fine with both.

Reply:
s. above